From 486e4ca33c3ea76848624063dd9426a0107dd06f Mon Sep 17 00:00:00 2001
From: Drew Moseley <drew_moseley@mentor.com>
Date: Sat, 5 Jul 2014 17:10:52 -0400
Subject: [PATCH 53/60] ACPI: Set acpi_enforce_resources to
 ENFORCE_RESOURCES_LAX

On some AMD platforms, the SMBus IO region 0xb00 - 0xb07 conflicts with the
corresponding ACPI SystemIO region. This prevents the SMBus host controller
driver to function correctly. We set acpi_enforce_resources to
ENFORCE_RESOURCES_LAX so that the SMBus driver can work normally, even
though a warning message notifying the conflict is printed in system logs.

Signed-off-by: Arindam Nath <arindam.nath@amd.com>
---
 drivers/acpi/osl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/acpi/osl.c b/drivers/acpi/osl.c
index e5f416c..cede198 100644
--- a/drivers/acpi/osl.c
+++ b/drivers/acpi/osl.c
@@ -1533,7 +1533,7 @@ __setup("acpi_serialize", acpi_serialize_setup);
 #define ENFORCE_RESOURCES_LAX    1
 #define ENFORCE_RESOURCES_NO     0
 
-static unsigned int acpi_enforce_resources = ENFORCE_RESOURCES_STRICT;
+static unsigned int acpi_enforce_resources = ENFORCE_RESOURCES_LAX;
 
 static int __init acpi_enforce_resources_setup(char *str)
 {
-- 
1.9.1

